src/tags/object/RichText/table.js

```diff
@@ -38,27 +57,112 @@ const renderTableValue = (val) => {
     return <div className={errClass}>{errMsg}</div>;
   }
 
-  const itemClass = cn('richtext', { elem: 'table-item' });
+  const questionItemClass = cn('richtext', { elem: 'table-item', mod: { qa : 'question' } });
+  const answerItemClass = cn('richtext', { elem: 'table-item', mod: { qa : 'answer' } });
+  let hasMath = false;
 
   const rowElems = conversations.map((conversation, index) => {
     const question = conversation[0];
     const answer = conversation[1];
+    const mathQuestions = parseConvoWithMath(question);
+    const mathAnswers = parseConvoWithMath(answer);
+    let mathQuestionComponent = null;
+    let mathAnswerComponent = null;
+
+    // Render an alternate list between Math and non-math expressions
+    const renderAllMathJax = (convoAndMathList) => (
+      convoAndMathList.map((convo, i) => {
+        if (i % 2 === 0) {
+          // Non math
+          return <span key={`eq=${i}`}>{convo}</span>;
```

Comment:
Sorry I think I'm missing something - how come every other one is not math?

Reply:
Should explain this better in comment - `parseConvoWithMath` return a list of Math / non-math expression alternatively, which we wraps around here.